build/gulp/tasks/perf.ts

```diff
@@ -82,39 +105,36 @@ const getPercentDiff = (minValue: number, actualValue: number): number =>
   _.round((actualValue / minValue) * 100 - 100, 2)
 
 const createMarkdownTable = (
-  normalizedMeasures: NormalizedMeasures,
-  metricName: string = 'actualTime',
-  fields: string[] = ['avg', 'median'],
+  perExamplePerfMeasures: PerExamplePerfMeasures,
+  metricName: MeasuredValues = 'actualTime',
+  fields: ('avg' | 'avgNormalized' | 'median' | 'medianNormalized')[] = [
```

Comment:
```suggestion
  fields: (key of Omit<ReducedMeasures, value>)[] = [
```

It looks like a valid change